### PR TITLE
Fix merging defaults and options

### PR DESCRIPTION
**Pull request by szmarczak**
Fixes commit 058452b.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Sorry, I moved the source files into a `source` directory earlier... Could you update the PR?

File: source/index.js [hunk 1]
```diff
@@ -654,10 +645,25 @@ function normalizeArguments(url, options) {
 	return options;
 }
 
+function assignOptions(defaults, options = {}) {
```
Comment (sindresorhus) [inline source/index.js]: The `= {}` is no longer needed here.

Comment (szmarczak) [reply source/index.js]: Nope. Tests have failed :laughing:

Comment (sindresorhus) [reply source/index.js]: Ugh, yeah I didn't notice `options` was used outside the `extend` call...

**Pull request merged**